### PR TITLE
tests, network: XFail IPv6 connectivity check to an external address

### PR DESCRIPTION
**Pull request by EdDev**
**What this PR does / why we need it**:
IPv6 connectivity check to the statically set address fails on CI.
The same tests pass locally.

Marking the assertion with xfail in order to allow investigation.

Ref: https://github.com/kubevirt/kubevirt/issues/5113

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
Seen on https://testgrid.k8s.io/kubevirt-presubmits#pull-kubevirt-e2e-k8s-cnao-1.19

Originally raised by @enp0s3 on slack.

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Review by enp0s3 — APPROVED** [review]
/approve

**Comment by EdDev** [comment]
If I understand correctly, this has been now addressed by https://github.com/kubevirt/project-infra/pull/987 and https://github.com/kubevirt/project-infra/pull/989 .

@fgimenez , @enp0s3 , @rmohr I'm placing this on hold for now.
/hold

**Comment by EdDev** [comment]
> If I understand correctly, this has been now addressed by [kubevirt/project-infra#987](https://github.com/kubevirt/project-infra/pull/987) and [kubevirt/project-infra#989](https://github.com/kubevirt/project-infra/pull/989) .
> 
> @fgimenez , @enp0s3 , @rmohr I'm placing this on hold for now.
> /hold

Based on #5126 run, the IPv6 issue is confirmed to be fixed by the bootstrap update.
Closing this PR.

**Comment by stu-gott** [comment]
/hold cancel

PR 5130 was closed -- this PR is still necessary short term.

**Comment by EdDev** [comment]
@stu-gott I understood the lanes are good now with the new bootstrap image, therefore this PR is not a must.

Do you see a specific problem that occurs now?

**Comment by stu-gott** [comment]
@EdDev I was just following up with the conversation in https://github.com/kubevirt/kubevirt/pull/5130 -- I understood it to mean this PR was still needed. You have more situational awareness on this topic than I do atm, so do what you think is right.

**Comment by EdDev** [comment]
> @EdDev I was just following up with the conversation in #5130 -- I understood it to mean this PR was still needed. You have more situational awareness on this topic than I do atm, so do what you think is right.

Ahh yes, it is a bit confusing.
Well, it goes like this: The latest bootstrap bump fixed the docker config and therefore allowed the tests in question to pass.
What was raised in that conversation was that the test itself (and actually another one) is problematic as it expects external cluster configuration (docker config in IPv6 case, internet access in IPv4 case), so we need to handle this to avoid failures when running on non-kubevirtci setups.

To summarize, we do not need to xfail/skip these now, we need to handle them in a different way later.
Said that, I do not mind if this is getting in until that happens.

I suggest to leave it to @AlonaKaplan to decide here.

**Comment by AlonaKaplan** [comment]
> > @EdDev I was just following up with the conversation in #5130 -- I understood it to mean this PR was still needed. You have more situational awareness on this topic than I do atm, so do what you think is right.
> 
> Ahh yes, it is a bit confusing.
> Well, it goes like this: The latest bootstrap bump fixed the docker config and therefore allowed the tests in question to pass.
> What was raised in that conversation was that the test itself (and actually another one) is problematic as it expects external cluster configuration (docker config in IPv6 case, internet access in IPv4 case), so we need to handle this to avoid failures when running on non-kubevirtci setups.
> 
> To summarize, we do not need to xfail/skip these now, we need to handle them in a different way later.
> Said that, I do not mind if this is getting in until that happens.
> 
> I suggest to leave it to @AlonaKaplan to decide here.

Lets keep the XFail here and add one to the test expects ipv4 internet access till we will introduce flags as suggested [here](https://github.com/kubevirt/kubevirt/pull/5130#pullrequestreview-601723234).
